### DISCUSSION TITLE
Duda con altura de los arboles

### DISCUSSION THREAD

**Discussion by gaspar-mr05** [IIC2133-PUC/2024-1#129] [Preguntas materia] [answered]
Hola! Tengo una duda muy simple respecto a la altura de los arboles. En algunas preguntas se considera que la altura = 0 se da cuando solo es un nivel, sin embargo en otros años (los mas recientes), consideran que el arbol de altura 0 es aquel que no tiene nodos, y que el que tiene un nivel es el de altura = 1. Como se debe considerar?

**Comment by Dafnemami** [comment, marked as answer]
Hola Gaspar!, la convención del curso al menos es que un árbol conformado solo por su raíz tiene solo un nivel (el `0`) y una altura `h=1`. Desde esa base construyes el resto. Niveles van desde `0` a `h-1`.
